src/documentation/layout-documentation.scss

```diff
@@ -0,0 +1,78 @@
+$header-height: 56px;
+$nav-width: 300px;
+
+.tikui-doc {
+  &-header {
```

Comment:
Maybe use the CAP style syntax (sorry it's in french but here is the reference: https://blog.ippon.fr/2020/05/04/definir-son-style-css-avec-cap/)